<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>udp_subs</class>
 <widget class="QDialog" name="udp_subs">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>646</width>
    <height>457</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QVBoxLayout" name="udp_subs_param">
     <item>
      <layout class="QHBoxLayout" name="master_ipv4" stretch="1,1">
       <item>
        <widget class="QLabel" name="ui_master_ipv4">
         <property name="text">
          <string>主机IPV4地址：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="ui_master_ipv4_in"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="master_port" stretch="1,1">
       <item>
        <widget class="QLabel" name="ui_master_port">
         <property name="text">
          <string>主机PORT端口：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="ui_master_port_in"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="slaver_ipv4" stretch="1,1">
       <item>
        <widget class="QLabel" name="ui_slaver_ipv4">
         <property name="text">
          <string>设备IPV4地址：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="ui_slaver_ipv4_in"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="slaver_port" stretch="1,1">
       <item>
        <widget class="QLabel" name="ui_slaver_port">
         <property name="text">
          <string>设备PORT端口：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="ui_slaver_port_in"/>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>udp_subs</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>udp_subs</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
